test/build-combination/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ set( TEST_DIR ${MODULE_ROOT_DIR}/test/build-combination )
 include_directories( ${MODULE_ROOT_DIR}/source/include )
 include_directories( ${MODULE_ROOT_DIR}/source/portable/Compiler/MSVC )
 include_directories( ${FREERTOS_KERNEL_DIR}/include )
-include_directories( ${FREERTOS_KERNEL_DIR}/portable/MSVC-MingW )
+include_directories( ${FREERTOS_KERNEL_DIR}/portable/ThirdParty/GCC/Posix )
```

Comment:
Does this change render the build useless on windows based platforms?

Reply:
Was it designed to be used on Windows? If so, should we not use Windows port on Windows and POSIX port on UNIX platforms?

Reply:
It is just a build check. It doesn't create a working application. It just makes sure that in the combinations there is no compilation error.
If at all possible, I would like if we can include the directories conditionally based on the underlying OS.